spec/requests/users_controller_spec.rb

```diff
@@ -31,6 +33,24 @@
         post api_v1_users_path, params: valid_user_params, headers: headers
         expect(User.find_by(email: valid_user_params[:user][:email]).role).to eq(role)
       end
+
+      context 'User language' do
+        it 'Persists the user language in the user record' do
+          expect { post api_v1_users_path, params: valid_user_params, headers: }.to change(User, :count).from(0).to(1)
+          expect(response).to have_http_status(:created)
+          expect(User.find_by(email: valid_user_params[:user][:email]).language).to eq('en')
+        end
+
+        it 'defaults user language to default_locale if the language isn\'t specified' do
+          allow(I18n).to receive(:default_locale).and_return(:gl)
+          expect(I18n).to receive(:default_locale)
+
+          valid_user_params[:user][:language] = nil
+          expect { post api_v1_users_path, params: valid_user_params, headers: }.to change(User, :count).from(0).to(1)
+          expect(response).to have_http_status(:created)
+          expect(User.take.language).to eq('gl')
```

Comment:
Same as above with the `User.find_by`

Reply:
and with the duplication
```
allow(I18n).to receive(:default_locale).and_return("test-language")
post api_v1_users_path, params: valid_user_params, headers:
expect(User.find_by(email: valid_user_params[:user][:email]).language).to eq('test-language')
```

Reply:
But, that means that we won't be ensuring an adequate response code and that the user in the database may not be created by the `post api_v1_users_path` or is it normal to be omit this?

Reply:
The other tests already cover the user creation case, so in this case, it makes more sense just to simplify it down to ensuring the language is set correctly